src/App.js

```diff
@@ -1,24 +1,14 @@
-import logo from './logo.svg';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import './App.css';
+import Greetings from './components/Greetings';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Greetings />} />
+      </Routes>
```

Comment:
- [ ] According to project requirements, the React app uses the URL and displays a random greeting. However, I don't get any greetings displayed on the UI except the title `<h1>My greetings</h1>` 👇🏻 . Please address this. 👍🏻 

![image](https://github.com/Chaseknis/hello_react_front-end/assets/79058364/0ee1f338-1ee2-4995-a0d4-7dff4df864c4)


- 🤔  And Also I'm getting this warning message from the front-end console. 👇🏻 

![image](https://github.com/Chaseknis/hello_react_front-end/assets/79058364/28ff58f7-d86a-466a-b704-d8d70d057304)

Reply:
Hello @BenMukebo the code is working fine on my end Please watch the **video** I added for reference

https://github.com/Chaseknis/hello_react_front-end/assets/97366651/33a6f153-c5a5-49f5-8fce-261c75b38008

Reply:
Hello @Chaseknis, Don't waste your time, if it's working fine, You may request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). so that the senior CR will approve this. 👍🏻